central/sensor/service/connection/connection_impl.go

```diff
@@ -72,6 +73,8 @@ type sensorConnection struct {
 	capabilities set.Set[centralsensor.SensorCapability]
 
 	hashDeduper hashManager.Deduper
+
+	maxSeenMessageSize map[string]float64
```

Comment:
I think we need to lock around access. This t thread safe

Reply:
This should've been removed, it's not used.